backend/src/apiserver/main.go

```
@@ -93,6 +95,7 @@ func main() {
 
 	options := &cm.Options{
 		UsePipelineKubernetesStorage: *usePipelinesKubernetesStorage,
+		GlobalKubernetesWebhookMode:  *globalKubernetesWebhookMode,
```

Do you think we should return an error if the webhook is disabled but in global Kubernetes webhook mode? It seems like an invalid state to run in.

---

Good point —I will update the code to return an error if the webhook is disabled while the global Kubernetes webhook mode is enabled.